packages/components/inputs/selectable-search-input/src/selectable-search-input.tsx

```diff
@@ -447,6 +453,11 @@ const SelectableSearchInput = (props: TSelectableSearchInputProps) => {
             textInputRef={textInputRef}
             selectedOption={selectedOption}
             dataProps={transformedSelectDataProps}
+            components={
+              {
+                ...props.components,
+              } as ReactSelectProps['components']
+            }
```

Comment:
Do we really need to spread and cast?

I'm asking because the prop is already typed as expected by the `SelectableSelect` component, right?
```suggestion
            components={props.components}
```

Reply:
updated hereL https://github.com/commercetools/ui-kit/pull/2831/commits/aa3cd25ca24a4ba0a42c335678e0d72d346457a3

Thank you